src/test/java/seedu/address/testutil/HeightBuilder.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.testutil;
+
+import seedu.address.model.person.Height;
+
+public class HeightBuilder {
```

Comment:
Is there a need for such a class? Seems to comprise of simple getters and setter.